spec/api-process-spec.js

```diff
@@ -67,4 +71,18 @@ describe('process module', () => {
       expect(heapStats.doesZapGarbage).to.be.a('boolean')
     })
   })
+
+  describe('process.takeHeapSnapshot()', () => {
+    it('returns true on success', (done) => {
+      const filePath = path.join(remote.app.getPath('userData'), 'test.heapsnapshot')
+      const success = process.takeHeapSnapshot(filePath)
```

Comment:
Can we check that the file is not empty?

Reply:
done